### ISSUE TITLE
Add focused block's icon to Quick Toolbar

### ISSUE THREAD

**Issue by iamthomasbishop** [wordpress-mobile/gutenberg-mobile#878]
## Current

![IMG_2653](https://user-images.githubusercontent.com/1200432/56243266-c97bbc80-605f-11e9-9117-51431b4d6237.png)

Currently, when a user is focused on a block, we aren't displaying the icon of that block in the Quick Toolbar, which means it's not always obvious what block is selected and why the tools change. AFAIK we aren't going to be building transforms right now, but we should at least display the current block icon as a helper/indicator.

## Proposed

<img width="521" alt="Screen Shot 2019-04-16 at 4 01 49 PM" src="https://user-images.githubusercontent.com/1200432/56243771-04cabb00-6061-11e9-8616-eba54e8021cf.png">

Note: ignore the differences in specific icons above. We're currently using Dashicons, so until we move away from that set, let's continue using Dashicons in practice.

**Comment by SergioEstevao** [comment]
@iamthomasbishop this a thing that we still want to do?

**Comment by iamthomasbishop** [comment]
@SergioEstevao Yes, we still need to do this. It’s less of an issue now that we have the floating toolbar, but it’d be nice to implement this for consistency and so we have it when we eventually implement block transforms.
